.github/workflows/ci.yml

```diff
@@ -20,6 +20,10 @@ jobs:
       - name: Install Yarn dependencies
         run: |
           yarn install
+      - name: Install Satyrographos dependencies
+        run: |
+          opam install satysfi-fonts-dejavu
```

Comment:
You can run the following line to install local satysfi-base with opam.
```sh
# if the directory is managed by VCS
opam pin add --verbose --yes "."
# otherwise
opam pin add --verbose --yes "file://$PWD"
```

Reply:
TIL, thank you!